draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -617,6 +691,7 @@
         faster on Site-A even if Site-A is overutilized.</t>
 
         <t><list style="hanging">
+	    <!-- XXX JMH - was there a particular intent on using a different bullet style here? -->
```

Comment:
no. The different bullet style was caused by different co-authors contributions.

Reply:
Next push will use "-".